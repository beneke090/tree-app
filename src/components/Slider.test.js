import React from 'react';
import renderer from 'react-test-renderer';
import Slider from './Slider';

describe('Slider', () => {
  test('should match snapshot.', () => {
    const component = renderer.create(
      <Slider label="testslider" min={0} max={100} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
