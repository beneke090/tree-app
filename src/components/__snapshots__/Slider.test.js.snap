// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider should match snapshot. 1`] = `
<label
  className="block mt-4"
  htmlFor="slider"
>
  <span
    className="text-gray-700"
  >
    testslider
  </span>
  <div
    id="slider"
  >
    <span>
      0%
    </span>
    <input
      max={100}
      min={0}
      onChange={[Function]}
      step="1"
      type="range"
    />
    <span>
      100%
    </span>
  </div>
</label>
`;
